python/src/diskann_bindings.cpp

```diff
@@ -204,7 +204,7 @@ template <class T> struct DynamicInMemIndex
         py::array_t<int> insert_retvals(num_inserts);
 
 #pragma omp parallel for schedule(dynamic, 1)
-        for (size_t i = 0; i < num_inserts; i++)
+        for (int64_t i = 0; i < num_inserts; i++)
```

Comment:
size_t causes a windows build error, using explicit type instead

Reply:
Then the argument passed to the function should be int64_t too